lib/include/openamp/rpmsg_virtio.h

```diff
@@ -41,10 +41,21 @@ struct rpmsg_virtio_shm_pool {
 	size_t size;
 };
 
+/**
+ * struct rpmsg_virtio_config - configuration of rpmsg device based on virtio
+ * @txbuf_size: the tx buffer size, used to send data from host to remote
+ * @rxbuf_size: the rx buffer size, used to send data from remote to host
+ */
+struct rpmsg_virtio_config {
+	uint32_t txbuf_size;
+	uint32_t rxbuf_size;
```

Comment:
can we use `size_t` for sizes instead?

Reply:
It derived from PR https://github.com/OpenAMP/open-amp/pull/155. How size_t map to the real built-in type(32bit v.s. 64bit) depend on the platform. So if you consider in a complex SoC: one side is 32bit MCU/DSP, another side is AArch/RISV64/X86, size_t isn't a good choice here.

Reply:
I agree with @xiaoxiang781216 it's not simple decision. The type should be selected based on the integration with users of this API. I would prefer to use `uint32_t` now, because it seems flexible and well-defined enough for most users. We can change the type later if we find other are more suitable.

Reply:
agreed